Sources/Views/Home.swift

```diff
@@ -167,39 +167,103 @@ private func episodesListView(episodes: ArraySlice<Episode>, emergencyMode: Bool
 
 private func episodeRowView(episode: Episode, emergencyMode: Bool) -> Node {
   @Dependency(\.date.now) var now
-  @Dependency(\.siteRouter) var siteRouter
 
   return [
     divider,
     .gridRow(
       .gridColumn(
         sizes: [.mobile: 12, .desktop: 7],
-        episodeInfoColumnView(episode: episode, emergencyMode: emergencyMode)),
+        episodeInfoColumnView(episode: episode, emergencyMode: emergencyMode)
+      ),
       .gridColumn(
         sizes: [.mobile: 12, .desktop: 5],
         attributes: [.class([Class.grid.first(.mobile), Class.grid.last(.desktop)])],
-        .div(
-          attributes: [
-            .class([Class.size.height100pct]),
-            .style(lineHeight(0) <> gradient <> minHeight(.px(300))),
-          ],
-          .a(
-            attributes: [.href(siteRouter.path(for: .episode(.show(.left(episode.slug)))))],
-            .img(
-              attributes: [
-                .src(episode.image),
-                .alt(""),
-                .class([Class.size.width100pct, Class.size.height100pct]),
-                .style(objectFit(.cover)),
-              ]
-            )
-          )
-        )
+        episodeImageColumnView(episode: episode)
       )
     ),
   ]
 }
 
+private func episodeImageColumnView(episode: Episode) -> Node {
+  @Dependency(\.episodeProgresses) var episodeProgresses
+  @Dependency(\.siteRouter) var siteRouter;
+
+  let brightness: Stylesheet
+  let watched: Node
+  if episodeProgresses[episode.sequence]?.isFinished == true {
+    if episode.sequence.rawValue.quotientAndRemainder(dividingBy: 4).remainder == 3 {
+      brightness = key("filter", "grayscale(1) brightness(175%)")
+    } else {
+      brightness = key("filter", "grayscale(1)")
+    }
```

Comment:
I have to do this funny thing to bump the brightness of every 4th image because the purple posters tend to be a little darker than the others.

Ideally we will separate the tint from all of our posters and apply the tints in CSS, but that's a lot of work.